Mathematical Analysis complete

Joseph-Burley opened the pull request:
The mathematical analysis section goes in depth on how to find the constants for the harmonics. The other section is a paragraph about how the labview file was set up and how we got the graphs. Feel free to make changes or suggestions.

Commit:
started working on lab2

Commit:
Started on mathematical analysis

Commit:
more work

Commit:
Finished mathematical analysis section

Commit:
Merge branch 'jmaster' into lab2-nh

theNerd247 commented:
Great work! I like the math derivations!

theNerd247 commented on lab2/mathematicalAnalysis.tex:
```diff
@@ -28,5 +28,29 @@
+	$$f(t) = \sum_{k=1}^{\infty} \frac{4}{(2k-1) \pi} sin\Big(\frac{2 \pi (2k-1)}{T}t \Big)$$
+	
+	
+	For $k=1$   $1.2732 sin(2 \pi f)$
```
What was the intention of this paragraph? Also was this supposed to be a table? If not the formatting is a bit off on my machine (the spacing between the constants and the sines are really crammed together.

Commit:
made some minor changes to part one draft

  * reformatted terms table
  * added equation and table references.
  * minor word changing.

Commit:
removed old part1 file and created template for part2

Commit:
almost done second part.

I'm not sure what the last couple of bullet points are in the
connections section.

Joseph-Burley commented:
The purpose of that paragraph was to show explicitly how and why I went from using n as the index to k. The portion where it says 
k=1 ...
k=2 ...
etc. 
is supposed to be a table, but I was writing that at 8:30 and didn't want to look up how to do a table.

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into theNerd247/master

theNerd247 commented:
Ok. I fixed it

On Thu, Sep 8, 2016, 10:16 AM Joseph Burley notifications@github.com
wrote:

> The purpose of that paragraph was to show explicitly how and why I went
> from using n as the index to k. The portion where it says
> k=1 ...
> k=2 ...
> etc.
> is supposed to be a table, but I was writing that at 8:30 and didn't want
> to look up how to do a table.
> 
> —
> You are receiving this because you were assigned.
> 
> Reply to this email directly, view it on GitHub
> https://github.com/theNerd247/mtre4100/pull/3#issuecomment-245611645,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AA3mj8_r7S11XtJKZu3ntnoRuXh1nP2Wks5qoBi7gaJpZM4J3gfZ
> .

Commit:
fgsdfgd

Commit:
Copied sty files for windows compatibility and fixed some typos

Commit:
Added abstract text

Added abstract text. Also changed equation 1 on page 3 to replace (1/T)
with f. Also added t to the equations in table 1 on same page.

Commit:
Merge pull request #1 from Joseph-Burley/lab2-nh

Lab2 nh

Commit:
Removed text command in abstract.
